app/src/main/java/com/github/fi3te/notificationcron/ui/NotificationCronDialog.kt

```diff
@@ -20,11 +21,12 @@ fun showCreateDialog(windowContext: Context, create: (notificationCron: Notifica
         addCronInputView(this)
 
         positiveButton(R.string.create) {
-            val (cron, notificationTitle, notificationText) = getInput(this)
+            val (cron, notificationTitle, notificationText, onClickURI) = getInput(this)
```

Comment:
For a better readability I would prefer onClickUri

Reply:
Do you think it would be better if instead of unpacking the data class I switch to using with block.

Something like:

```
with (getInput(this)) {
    val notificationCron = NotificatioCron(
        ...
    )
    create(notificationCron)
}
```

Reply:
For now I've made the change but do let me know what you think about using with block.

Reply:
I have no preference in this respect (too little experience with the Kotlin programming language)

Reply:
If you think that refactoring work is necessary and you are interested, you are welcome to create a PR 😄